src/readline/lib_readline.cr

```diff
@@ -0,0 +1,19 @@
+@[Link("readline")]
+{% if flag?(:openbsd) %}
+  @[Link("termcap")]
+{% end %}
+lib LibReadline
```

Comment:
So this is reverting https://github.com/crystal-lang/crystal/pull/8364 as is. So there are a couple of places we might need to update the required libraries.

Maybe we should hide this inside `Crystal::` namespace, since we probably don't aim to do a full binding at least for now.

Reply:
Yes, after I sent the PR I updated the description, check the section "About FFI, dlopen and readline". I'll move these into the Crystal namespace 👍

Reply:
Done!